test/puppetlabs/puppetdb/testutils/reports.clj

```diff
@@ -130,13 +101,19 @@
        (into (omap/ordered-map))))
 
 (defn munge-report
-  "Munges a catalog of list of reports for comparision.
-  Returns a list of reports."
-  [report-or-reports]
-  (->> report-or-reports
-       utils/vector-maybe
-       (map (comp munge-report-for-comparison
-                  munge-example-report-for-storage))))
+  [report]
+  (-> report
+      keywordize-keys
+      (update-in [:resource_events :data]
+                 (comp (partial sort-by
+                                #(mapv % [:timestamp :resource_type :resource_title :property]))
+                       (partial map
+                                (fn [resource-event]
+                                  (update resource-event :timestamp time-coerce/to-string)))))
```

Comment:
the last few lines here seem a bit hairier than what we had. splitting out a couple more functions might make it clearer